client/src/screens/events/EventUsers.js

```diff
@@ -59,7 +59,7 @@ class EventUsers extends Component {
             break;
           case 'attending':
             tempResponse.etaText = tempResponse.eta === 0 ? '' : `- ETA ${moment.unix(tempResponse.eta).fromNow()}`;
-            tempResponse.statusText = tempResponse.detail === ''
+            tempResponse.statusText = !tempResponse.detail || tempResponse.detail === ''
```

Comment:
Wasn't this in the other PR?

Reply:
weird. must have sucked in an old commit